src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -349,25 +353,16 @@ def __validate_number_of_clbits(self, experiment: QasmQobjExperiment) -> None:
 
         :raises QiskitBackendError: When the value is not correct.
         """
-        number_of_clbits = experiment.header.memory_slots
-        if number_of_clbits < 1:
-            raise QiskitBackendError("Invalid amount of classical bits ({})!".format(number_of_clbits))
-
-        measurements = self._collect_measurements(experiment)
-        number_of_classical_bits = measurements['number_of_clbits']
-        max_measurement_index = max(measurement[1] for measurement in measurements['measurements'])
-        if max_measurement_index >= number_of_classical_bits:
-            raise QiskitBackendError(f"Number of classical bits ({number_of_classical_bits}) is not sufficient for "
-                                     f"storing the outcomes of the experiment")
-
-        if Backend.configuration(self).conditional:
-            number_of_qubits = experiment.header.n_qubits
-            if number_of_clbits > number_of_qubits:
-                # no problem when there are no conditional gate operations
-                for instruction in experiment.instructions:
-                    if hasattr(instruction, 'conditional'):
-                        raise QiskitBackendError("Number of classical bits must be less than or equal to the"
-                                                 " number of qubits when using conditional gate operations")
+        header = experiment.header
+        number_of_qubits = header.n_qubits
+        number_of_clbits = header.memory_slots
+
+        if number_of_clbits > number_of_qubits:
+            for instruction in experiment.instructions:
+                if hasattr(instruction, 'conditional'):
+                    # no problem when there are no conditional gate operations
+                    raise QiskitBackendError("Number of classical bits must be less than or equal to the"
```

Comment:
nit: elsewhere in the file we seem to prefer single quotes for strings, also when raising exceptions (see for example line 332).

Reply:
ok